modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -472,11 +467,9 @@ private CPDefinition _addOrUpdateProduct(Product product) throws Exception {
 
 		serviceContext.setAssetTagNames(assetTagNames);
 
-		Map<String, Serializable> expandoBridgeAttributes =
-			_getExpandoBridgeAttributes(product);
-
-		if (expandoBridgeAttributes != null) {
-			serviceContext.setExpandoBridgeAttributes(expandoBridgeAttributes);
+		if (_getExpandoBridgeAttributes(product) != null) {
```

Comment:
@JanisZhang The commit says "won't return null" but the null check is still here. Did you forget to remove it?

Reply:
@dantewang SORRY.